package.json

```diff
@@ -79,12 +84,14 @@
     "less": "^3.9.0",
     "less-loader": "^4.0.4",
     "mini-css-extract-plugin": "^0.5.0",
+    "mocha": "^6.0.2",
     "postcss-base64": "^0.4.1",
     "postcss-custom-media": "^5.0.1",
     "postcss-loader": "^2.0.6",
     "postcss-url": "^7.0.0",
     "regenerator-runtime": "^0.12.1",
     "rimraf": "^2.5.4",
+    "selenium-webdriver": "^4.0.0-alpha.1",
```

Comment:
Using "alpha" dependency seems a bit risky, if everything works fine with stable previous release, I'd recommend on using that one instead.

Reply:
I think it's a stable release https://github.com/SeleniumHQ/selenium/issues/5617 🤔 pretty poor naming if so, though

Reply:
> We use semver b/c that's what npm uses, but it's not an accurate representation of the stability of each selenium-webdriver release. Every release pushed to npm is considered the latest stable supported release that users should consume.

I don't get the reasoning, but seems (s)he suggests to use it

Reply:
🤔 Well, if it's stable... then fine by me 👍